setup.py

```diff
@@ -11,7 +11,7 @@
 
 requirements = [
     # Until COMPAS reaches 1.0, we pin major.minor and allow patch version updates
-    'compas>=0.16.4,<0.17',
+    'compas>=0.16.4,<0.17',  # !!! must change
```

Comment:
compas>=0.16.8

Reply:
👍  `compas 0.16.8` is now on pip and conda